probes/hasGitHubWorkflowPermissionUndeclared/def.yml

```diff
@@ -12,15 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-id: gitHubWorkflowPermissionsTopNoWrite
-short: Checks that GitHub workflows do not have default write permissions
+id: hasGitHubWorkflowPermissionUndeclared
```

Comment:
this sounds like it should be inverted? since outcome positive is currently being returned for workflows where there arent undeclared permissions?

has no undeclared permissions?

Reply:
Removed the probe in https://github.com/ossf/scorecard/pull/3816/commits/3d0d1df221b03e0747e7d8b68291ead0856c06aa